storagetransfer/azure_request.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Command-line sample that creates a one-time transfer from Azure Blob Storage to
+Google Cloud Storage.
+"""
+
+
+import argparse
+
+# [START storagetransfer_transfer_from_azure]
+from datetime import datetime
+
+from google.cloud import storage_transfer
+
+
+def create_one_time_azure_transfer(
+        project_id: str, description: str,
+        azure_storage_account: str, azure_sas_token: str,
+        source_container: str, sink_bucket: str):
+    """Creates a one-time transfer job from Azure Blob Storage to Google Cloud
+    Storage."""
+
+    client = storage_transfer.StorageTransferServiceClient()
+
+    # The ID of the Google Cloud Platform Project that owns the job
+    # project_id = 'my-project-id'
+
+    # A useful description for your transfer job
+    # description = 'My transfer job'
```

Comment:
Is description really a required value or can it be hardcoded? https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments

Reply:
While not required, it is a best practice to have a name or description for uniquely identifying transfer jobs. I selected description to avoid any potential name collisions